sdcm/sct_config.py

```diff
@@ -931,6 +931,8 @@ class SCTConfiguration(dict):
                      When set to False the test will continue to run even when there are errors in the
                      stress process"""),
 
+        dict(name="use_legacy_cluster_init", env="SCT_USE_RECOMMEND_CLUSTER_INIT", type=bool,
+             help="""Init cluster in recommended way"""),
```

Comment:
why is it called legacy if it is recommended way to boot the cluster up?

Reply:
@bentsi the logic was switched by @enaydanov request. Legacy is our current method of cluster initialization. env has a mistake. Fixed.

Reply:
@aleksbykov 

A variable named
```
use_legacy_cluster_init
```
should not be called
```
SCT_USE_RECOMMENDED_CLUSTER_INIT
```
in environment.

Either use `SCT_USE_LEGACY_CLUSTER_INIT` in env, or `use_recommended_cluster_init` as the name.

Also, the help string should say what the recommended way is.

Reply:
@kbr- , yes, i already fixed. Thanks

Reply:
Thanks.